extending-valtimo/form-flow/whitelist-spring-bean.md

```diff
@@ -0,0 +1,26 @@
+# Whitelisting Spring beans
+
+Spring Expression Language (SpEL) by default allows access to Spring beans, e.g. to retrieve data from
+an external source. Valtimo uses a whitelist for this instead. In order to add a Spring bean to this
```

Comment:
Do we want to explain why we use a whitelist? Also, it's not really using a whitelist instead, because it still allows access to spring beans, just not all of them. But i'm not sure we need to change this description.

Reply:
I can add a quick description as to why we use a whitelist instead.